implementations/Golang_github.com-vmware-labs-yaml-jsonpath/main.go

```diff
@@ -0,0 +1,37 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: GPL-3.0
+
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+
+	"example.com/main/internal/wrapper"
+)
+
+func main() {
+	selector := os.Args[1]
+
+	data, err := ioutil.ReadAll(os.Stdin)
+	if err != nil {
+		fmt.Fprintln(os.Stderr, err)
+		os.Exit(1)
+	}
+
+	jsonResult, errType, err := wrapper.Find(selector, data)
+	if errType != wrapper.Ok {
+		fmt.Fprintln(os.Stderr, err)
+		switch errType {
+		case wrapper.NotSupported:
+			os.Exit(2) // not supported
+		case wrapper.NotFound:
+			os.Exit(1) // not found
```

Comment:
Got it the wrong way I think?

3 is not found (https://github.com/cburgmer/json-path-comparison/blob/master/src/query_implementation.sh#L64), 1 is general error.

Reply:
Ah, my mistake. I'll fix that now.